applications/serial_lte_modem/boards/nrf9160dk_nrf9160_ns.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration file for nRF9160DK.
+# This file is merged with prj.conf in the application folder, and options
+# set here will take precedence if they are present in both files.
+
+# Use UART_0 (when working with PC terminal)
+# unmask the following config
+CONFIG_SLM_CONNECT_UART_0=y
+CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
+CONFIG_UART_0_NRF_HW_ASYNC=y
+CONFIG_SLM_INTERFACE_PIN=6
+
+# Use UART_2 (when working with external MCU)
+# unmask the following config
+#CONFIG_SLM_CONNECT_UART_2=y
+#CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
+#CONFIG_UART_2_NRF_HW_ASYNC=y
+#CONFIG_SLM_INTERFACE_PIN=31
```

Comment:
I think it would be better to place this in a dedicated file, for example:
`applications/serial_lte_modem/nrf9160dk_external_mcu.conf`
```
# This file supports working with external MCU
CONFIG_SLM_CONNECT_UART_2=y
CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
CONFIG_UART_2_NRF_HW_ASYNC=y
CONFIG_SLM_INTERFACE_PIN=31
```
and then a user can select this, by doing `west build .... -- -DOVERLAY_CONFIG=nrf9160dk_external_mcu.conf`

And not having to modify  `nrf9160dk_nrf9160_ns.conf`

Note, consider if some of the settings from `nrf9160dk_nrf9160_ns.conf` should be adjusted, for example:
```
# nrf9160dk_external_mcu.conf
...
# Disable slm connect uart 0 when using external mcu
CONFIG_SLM_CONNECT_UART_0=y
```